spec/index.bs

```diff
@@ -938,6 +940,23 @@ dictionary GPULimits {
         across all {{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
         when creating a {{GPUPipelineLayout}}.
 
+        Higher is [=better=].
+
+    : <dfn>maxTextureSize</dfn>
+    ::
+        The maximum size in a single dimension of a texture.
+
+        Note: This is only used in the width dimension for 1D textures, and the width & height dimensions for 2D textures.
+              This isn't used at all for 3D textures.
+
+        Higher is [=better=].
+
+    : <dfn>maxTextureLayers</dfn>
+    ::
+        The maximum number of layers in an array texture.
+
+        Note: This is also used for the size limit of 3D textures.
```

Comment:
All three dimensions are tied to maxTextureLayers?

Is there a fundamental reason this is true (i.e. coming from the native APIs)? If not, we should probably split up the limits in case there is hardware out there where this would be overly restrictive.

Also would you mind updating https://github.com/gpuweb/gpuweb/blob/master/design/Limits.md to document where these numbers come from?